src/NoFrixion.MoneyMoov/Models/Rules/Rule.cs

```diff
@@ -61,6 +61,23 @@ public string GeApprovalHash()
         return HashHelper.CreateHash(input);
     }
 
+    private bool IsCronExpressionScheduledMoreThanXMinutes(string cronExpression, int minutes)
+    {
+        var interval = TimeSpan.FromMinutes(minutes);
+
+        // Parse the cron expression
+        var cron = new CronExpression(cronExpression);
+
+        // Get the next 2 scheduled dates/times for the cron expression
+        var now = DateTime.Now;
+        var next1 = cron.GetNextValidTimeAfter(now);
+        var next2 = cron.GetNextValidTimeAfter(next1 ?? now);
+
+        // If the interval between the next 2 scheduled dates is less than X minutes,
+        // the cron expression will run more frequently than every X minutes
+        return (next2 - next1) >= interval;

```

Comment:
This will be true if the cron expression is 15mins and in your check it will fail. Maybe just greater(>) instead on greater and equal.